### DISCUSSION TITLE
GEO Location Issue

### DISCUSSION THREAD

**Discussion by extremeleadprogram** [YOURLS/YOURLS#3193] [General]
The locations seem to be starting to show countries outside of the US but when I manually check the IP's with Maxmind they are from the US.

I am using 1.7.9, should I update or would using the GEO plugin be the best way to get more accurate GEO data in the stats?

**Comment by ozh** [comment]
Yeah, the DB shipped with YOURLS is old. I've updated it, see #3191

**Comment by extremeleadprogram** [comment]
Ok, that makes sense. I also have a paid license to MaxMind, can I use the paid version and how would I do that the easiest?

**Reply by ozh** [reply]
The "easiest" : simply replace the `.mmdb` file - but it will be overwritten with next YOURLS release

The best : make a simple plugin to point to a user defined `.mmdb`, see [this answer](https://github.com/YOURLS/YOURLS/issues/3173#issuecomment-1003085921)
